packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -23,6 +23,23 @@ export interface IService extends IResource {
   readonly serviceArn: string;
 }
 
+export interface EcsTarget {
+  /**
+   * Properties for defining an ECS target.
+   */
+  readonly containerTarget: LoadBalancerTargetOptions,
+
+  /**
+   * ID for a target group to be created.
+   */
+  readonly targetGroupId: string,
```

Comment:
I would prefer you called this `newTargetGroupId` to make it abundantly clear you will be creating a new TargetGroup for users, rather than picking an existing Target Group that they've already created.